snuba/admin/static/tracing/index.tsx

```diff
@@ -1,8 +1,115 @@
-import React from "react";
+import React, { useState } from "react";
 import Client from "../api_client";
 import QueryDisplay from "../components/query_display/query_display";
 import { QueryResult } from "../components/query_display/types";
 
+type LogLine = {
```

Comment:
Could you put all of this logic in a different file inside the tracing/ directory? This file will only get more complex as we change the UI so I'd like to keep things separate if possible.